js/widgets/jseditor.js

```diff
@@ -377,6 +377,12 @@ class JSEditor {
         this._code = JSGenerate.code;
         this._jar.updateCode(this._code);
         this._setLinesCount(this._code);
+        const helpBtn = docById("js_editor_help_btn");
+        if(helpBtn) {
```

Comment:
space before the ( please.

Reply:
@walterbender made changes as recommended